docs/source/user-guide/sql/sql_status.md

```diff
@@ -34,106 +34,69 @@
-    - [x] [to_timestamp_micros](./scalar_functions.md#to_timestamp_micros)
-    - [x] [to_timestamp_seconds](./scalar_functions.md#to_timestamp_seconds)
-    - [x] [extract](./scalar_functions.md#extract)
-    - [x] [date_part](./scalar_functions.md#date_part)
-- nested functions
-  - [x] Array of columns
+- [x] [String Functions](./scalar_functions.md#string-functions)
+- [x] [Conditional Functions](./scalar_functions.md#conditional-functions)
+- [x] [Time and Date Functions](./scalar_functions.md#time-and-date-functions)
+- [x] [Math Functions](./scalar_functions.md#math-functions)
+- [x] [Aggregate Functions](./aggregate_functions.md) (`SUM`, `MEDIAN`, and many more)
 - [x] Schema Queries
-  - [x] SHOW TABLES
-  - [x] SHOW COLUMNS FROM <table/view>
-  - [x] SHOW CREATE TABLE <view>
-  - [x] information_schema.{tables, columns, views}
-  - [ ] information_schema other views
-- [x] Sorting
-- [ ] Nested types
-- [ ] Lists
+  - [x] `SHOW TABLES`
+  - [x] `SHOW COLUMNS FROM <table/view>`
+  - [x] `SHOW CREATE TABLE <view>`
+  - [x] Basic SQL [Information Schema](./information_schema.md) (`TABLES`, `VIEWS`, `COLUMNS`)
+  - [ ] Full SQL [Information Schema](./information_schema.md) support
+- [ ] Support for nested types (`ARRAY`/`LIST` and `STRUCT`. See [#2326](https://github.com/apache/arrow-datafusion/issues/2326) for details)
+  - [x] Read support
+  - [x] Write support
+  - [x] Field access (`col['field']` and [`col[1]`])
+  - [x] [Array Functions](./scalar_functions.md#array-functions)
+  - [ ] [Struct Functions](./scalar_functions.md#struct-functions)
+    - [x] `struct`
+    - [ ] [Postgres JSON operators](https://github.com/apache/arrow-datafusion/issues/6631) (`->`, `->>`, etc.)
 - [x] Subqueries
-- [x] Common table expressions
-- [x] Set Operations
-  - [x] UNION ALL
-  - [x] UNION
-  - [x] INTERSECT
-  - [x] INTERSECT ALL
-  - [x] EXCEPT
-  - [x] EXCEPT ALL
-- [x] Joins
-  - [x] INNER JOIN
-  - [x] LEFT JOIN
-  - [x] RIGHT JOIN
-  - [x] FULL JOIN
-  - [x] CROSS JOIN
-- [ ] Window
-  - [x] Empty window
-  - [x] Common window functions
-  - [x] Window with PARTITION BY clause
-  - [x] Window with ORDER BY clause
-  - [ ] Window with FILTER clause
-  - [ ] [Window with custom WINDOW FRAME](https://github.com/apache/arrow-datafusion/issues/361)
-  - [ ] UDF and UDAF for window functions
+- [x] Common Table Expressions (CTE)
+- [x] Set Operations (`UNION [ALL]`, `INTERSECT [ALL]`, `EXCEPT[ALL]`)
+- [x] Joins (`INNER`, `LEFT`, `RIGHT`, `FULL`, `CROSS`)
+- [x] Window Functions
+  - [x] Empty (`OVER()`)
+  - [x] Partitioning and ordering: (`OVER(PARTITION BY <..> ORDER BY <..>)`)
+  - [x] Custom Window (`ORDER BY time ROWS BETWEEN 2 PRECEDING AND 0 FOLLOWING)`)
+  - [x] User Defined Window and Aggregate Functions
+- [x] Catalogs
+  - [x] Schemas (`CREATE / DROP SCHEMA`)
+  - [x] Tables (`CREATE / DROP TABLE`, `CREATE TABLE AS SELECT`)
+- [ ] Data Insert
+  - [x] `INSERT INTO`
```

Comment:
Sorry @alamb that is not very clear to me.
Do you mean DF supports SQL syntax to insert data into CSV file?

Reply:
Yes, that is correct Here is an example:

```shell
(arrow_dev) alamb@MacBook-Pro-8:~/Software/arrow-site$ touch /tmp/foo.csv
(arrow_dev) alamb@MacBook-Pro-8:~/Software/arrow-site$ datafusion-cli
DataFusion CLI v26.0.0
❯ create external table foo (x int) stored as csv location '/tmp/foo.csv';
0 rows in set. Query took 0.004 seconds.
❯ insert into foo values (1), (2), (3);
+-------+
| count |
+-------+
| 3     |
+-------+
1 row in set. Query took 0.005 seconds.
❯
\q
(arrow_dev) alamb@MacBook-Pro-8:~/Software/arrow-site$ cat /tmp/foo.csv
1
2
3
```

The UX is somewhat awkward at the moment as it can only append into existing files (not create new ones). I have real hopes it will be improved

Reply:
Oh that is really cool, I want to make this happen for parquet asap. Will create a ticket and implement it. Should be straightfwd as we have parquetwriter

Reply:
❤️  -- I also think https://github.com/apache/arrow-datafusion/issues/5654 is very related (rather than having to define the table in order to add to it). See #6539 where I started the basics